.editorconfig

```diff
@@ -0,0 +1,27 @@
+; EditorConfig to support per-solution formatting.
```

Comment:
Adding an .editorconfig sounds like a change that is out of scope for this PR.

Switching to an .editorconfig sounds like a decision/discussion we should have across all Bearded libraries.

Reply:
Without it the auto-formatting doesn't work in visual studio, but I'll remove it.

Reply:
It's a fair point, and something we should do a better job of.

Filed beardgame/utilities#212 to track this across all Bearded libraries.